app/server/index.js

```diff
@@ -174,10 +172,13 @@ app.post('/twilio', function(req, res) {
                                 })
                                 .catch(function(error){
                                     console.log("Storage FAILURE: ", error);
-                                    res.set('Content-Type', 'text/xml');
-                                    res.send(xml({"Response":""}));
+                                    res.sendStatus(422);
```

Comment:
422? It's supposed to be reserved for WebDAV. Let's do 500 in these cases (and send an expanatory message alongside in the body)
`res.status(500).send('failure because x, y, z')`